Data/Input/default.xci

```diff
@@ -390,7 +390,7 @@ type=key
 data=0
 event=Setup Alternates
 event=Mode default
-label=Alternates$(CheckWaypointFile)
+label=Alternates\nList$(CheckWaypointFile)
```

Comment:
I don't think adding "List" improves anything here.
There's not even a verb - note I don't think that all buttons should have a verb - but it's not even consistent with the policy this PR attempts to meet (a policy which I don't like).